src/main/scala/viper/silver/parser/ParseAst.scala

```diff
@@ -1298,6 +1283,15 @@ object PScope {
   }
 }
 
+// Macros
+sealed trait PMacro extends PExp {
```

Comment:
PMacroCall?

Reply:
I agree that the name should be changed, but this trait is also for non-call like macro refs (`PIdnUse`)